python_scripts/linear_models.py

```diff
@@ -565,9 +572,13 @@ def f(x):
 # the regressor, scikit-learn provides a `Pipeline`, which 'chains' the
 # transformer and regressor together. The pipeline allows you to use a
 # sequence of transformer(s) followed by a regressor or a classifier, in one
-# call. This pipeline exposes the same API as the regressor and classifier
+# call. (i.e. fitting the pipeline will fit both the transformer(s) and the regressor. 
+# Then predicting from the pipeline will first transform the data through the transformer(s)
+# then predict with the regressor from the transformed data)
```

Comment:
Calling `fit` on the pipeline is equivalent to calling `fit` all transformers and the regressor. Calling 

I also think the pipeline is an important concept and could/should be introduced in it's own section instead of tagged on (@GaelVaroquaux mentioned, pipelines are something people struggle to use and this MOOC might be a nice place to do a good intro to pipeline?). We introduce the pipeline first in basic preprocessing notebook. Maybe basic preprocessing would be a good place to have a longer introduction of pipelines in the basic preprocessing notebook (see https://github.com/INRIA/scikit-learn-mooc/pull/37#discussion_r469417632), as it seems to fit there.